test/benchmark.py

```diff
@@ -62,15 +62,32 @@ def human_int(value):
     return int(value) * multiplier
 
 
-def create_test_file(path, max_size, verbose=False):
+def create_test_file(path, max_size, verbose=False, chunk_size=65535):
+    if max_size < 1:
+        raise ValueError("max_size must be >= 1")
```

Comment:
The codebase uses single ticks for strings whenever it is possible without complicating things.

```suggestion
        raise ValueError('max_size must be >= 1')
```